.changeset/fuzzy-spiders-film.md

```diff
@@ -0,0 +1,5 @@
+---
+'@keystone-next/fields': minor
+---
+
+Added `json` field to `@keystone-next/fields` package.
```

Comment:
The user is reading a changelog in the @keystone-next/fields package so they already know where it's been added to.

```suggestion
Added `json` field.
```